molecule/Dockerfile-rhel9-java8.j2

```diff
@@ -37,10 +37,8 @@ RUN yum -y install java-1.8.0-openjdk \
       krb5-workstation \
       unzip \
       procps \
-      procps-ng
-
-# Workaround to fix GPG signature validation issue, remove when it is fixed in CP 7.5 release
-RUN update-crypto-policies --set DEFAULT:SHA1
```

Comment:
why did we remove this? Have we tested that fips+java8 works without this workaround?

Reply:
In oracle 8 when we have this line it gives the error [jenkins](https://jenkins.confluent.io/job/cp-ansible-on-demand/2199/testReport/molecule.rbac-mtls-rhel8.molecule/yml/test/) Unknown policy SHA1. Removing it makes the scenario run fine. 
This combination of fips + java8 is not tested in any scenarios. I can modify one scenario to check this works fine without the SHA1 policy

Reply:
agreed, I think we don't even support fips on rhel9 with java8. It's only supported on java 11 and java 17. If removing this line works fine, I think it's good to remove